v21.1/schema-design-overview.md

```diff
@@ -1,7 +1,112 @@
+
+For guidance on creating databases, see [Create a Database](schema-design-database.html).
+
+{{site.data.alerts.callout_info}}
+To avoid confusion with the general term "[database](https://en.wikipedia.org/wiki/Database)", throughout this guide we refer to the logical object as a *database*, to CockroachDB by name, and to a deployment of CockroachDB as a [*cluster*](architecture/overview.html#terms).
+{{site.data.alerts.end}}
+
+### Schemas
+
+Schemas make up the second level of the naming hierarchy. Each schema belongs to a single database. Schemas contain [tables](#tables), [indexes](#indexes), and other objects, like [views](#views) and [sequences](#sequences).
+
+All CockroachDB clusters include a preloaded schema named `public`. Rather than using the `public` schema, we recommend creating your own *user-defined schema*.
+
+For guidance on creating user-defined schemas, see [Create a User-defined Schema](schema-design-schema.html).
+
+{{site.data.alerts.callout_info}}
+To avoid confusion with the general term "[schema](https://en.wiktionary.org/wiki/schema)", in this guide we refer to the logical object as a *user-defined schema*, and to the relationship structure of logical objects in a cluster as a *database schema*.
+{{site.data.alerts.end}}
+
+### Tables
+
+Tables, belong to the third and lowest level of the naming hierarchy. Each table can belong to a single [user-defined schema](#schemas).
+
+Tables contain *rows* of data. Each value in a row of data belongs to a particular *column*. Each column allows values of data of a single data type. Columns can be further qualified with [column-level constraints](column-constraints.html), or computed with [scalar expressions](computed-columns.html).
+
+For guidance on defining tables, see [Tables](schema-design-tables.html).
+
+### Indexes
+
+Indexes, like tables, belong to a single user-defined schema, in the third level of the naming hierarchy.
+
+An index is a copy of the rows in a single table, sorted by a column or set of columns. CockroachDB queries use indexes to more efficiently find data in a table, given the rows values of a particular column.
+
+The two main types of indexes are the `primary` index, an index on the row-identifying [primary key columns](primary-key.html), and the secondary index, an index created on columns of your choice.
+
+For guidance on defining primary keys, see [Select Primary Key Columns](schema-design-tables.html#select-primary-key-columns). For guidance on defining secondary indexes, see [Add a Secondary Index](schema-design-indexes.html).
+
+#### Specialized indexes
+
+CockroachDB supports some specialized types of indexes, designed to improve query performance in specific use cases. For guidance on specialized indexes, see [Index a Subset of Rows](partial-indexes.html) and [Index Sequential Keys](hash-sharded-indexes.html).
```

Comment:
I'm not totally sure where inverted indexes figure into this discussion, but just want to note them.